src/smsfusion/_ins.py

```diff
@@ -763,26 +815,23 @@ def update(
         if degrees:
             w_imu = (np.pi / 180.0) * w_imu
 
-        # Update current state estimate
-        self._x = self._ins.x
+        # Current INS state estimates
+        pos_ins = self._ins._pos
+        vel_ins = self._ins._vel
+        q_nm_ins = self._ins._q_nm
```

Comment:
```suggestion
        q_ins_nm = self._ins._q_nm
```

We should prefer this notation -> q_ins_nm. The transformation subscript should come last so that its easier to follow transformations in code. Eg. 

`R_ins_nm * v_m`  instead of `R_nm_ins * v_m`